.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+To help us debug your issue please explain:
+
```

Comment:
Maybe we could add some fixed data to provide with like, conan --version, description of problem, (expected behaviour?), traceback.

Also optional things like os, compiler... may be of interest

Reply:
Is not covered by ` I've specified the Conan version, operating system version and any tool that can be relevant.` ?

Reply:
Ah, you mean like a section to fill. Not sure. let's see how it works, sometime it has no sense and it can be annoying to the user

Reply:
fine!